meio.opam

```
@@ -1,20 +1,20 @@
-# This file is generated by dune, edit dune-project instead
 opam-version: "2.0"
 synopsis: "Monitor live Eio programs"
 description:
   "Eio-console provides an executable that allows you to monitor OCaml programs using Eventring."
 maintainer: ["patrick@sirref.org"]
 authors: ["Patrick Ferris"]
 license: "MIT"
-homepage: "https://github.com/patricoferris/eio-console"
-bug-reports: "https://github.com/patricoferris/eio-console/issues"
+homepage: "https://github.com/tarides/meio"
```

The repo moved since this PR :)) 

```suggestion
homepage: "https://github.com/ocaml-multicore/meio"
```